code/datums/wounds/_wounds.dm

```diff
@@ -340,6 +340,16 @@
 /datum/wound/proc/drag_bleed_amount()
 	return
 
+/**
+ * get_bleed_rate_of_change() is used in [/mob/living/carbon/proc/bleed_warn] to gauge whether this wound (if bleeding) is becoming worse, better, or staying the same over time
+ *
+ * Returns BLOOD_FLOW_STEADY if we're not bleeding or there's no change (like piercing), BLOOD_FLOW_DECREASING if we're clotting (non-critical slashes, gauzed, coagulant, etc), BLOOD_FLOW_INCREASING if we're opening up (crit slashes/herapin)
+ */
+/datum/wound/proc/get_bleed_rate_of_change()
+	if(blood_flow && victim.reagents.has_reagent(/datum/reagent/toxin/heparin))
```

Comment:
Heparin should be giving a trait/status effect instead of checking it everywhere like this.

Reply:
Pretend I commented this on all your explicit heparin checks.